src/main/java/org/everit/jira/timetracker/plugin/Hash.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2011 Everit Kft. (http://www.everit.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.everit.jira.timetracker.plugin;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * The hash class.
+ */
+public final class Hash {
+
+  private static final int HEX_0X0F = 0x0F;
+
+  private static final int SHIFT_4 = 4;
+
+  private static final int HEX_0X_FF = 0xFF;
+
+  /**
+   * Converts byte array to hex String.
+   *
+   * @param bytes
+   *          The byte array.
+   * @return The hex value in string.
+   */
+  public static String bytesToHex(final byte[] bytes) {
+    final char[] hexArray = { '0', '1', '2', '3', '4', '5', '6', '7', '8',
```

Comment:
Do not use the final modifier for local variables, please fix your IDE settings. All of the modified classes must be fixed.